### PR TITLE
Cleaned up code and documentation to match SQS library.

### PR DESCRIPTION
**Pull request by dbns97**

### CODE REVIEW

**Comment by dbns97** [comment]
Hey @ppetrosh, I can't assign it to you but here's the pull request with our changes.

File: tests/agent.test.nut [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO
+// EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+// OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+// Please enter your AWS keys, region and SNS Topic ARN
+const AWS_SNS_TEST_REGION = "YOUR_REGION_HERE";
```
Comment (ppetrosh) [inline tests/agent.test.nut:26]: This should actually be env parameters

File: README.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+To add this library to your model, add the following lines to the top of your agent code:
+
+```
+#require "AWSRequestV4.class.nut:1.0.2"
+#require "AWSSNS.agent.lib.nut:1.0.0"
+```
+
+**Note: [AWSRequestV4](https://github.com/electricimp/AWSRequestV4/) must be
+included before the AWSSQS library to make it work.**
```
Comment (betzrhodes) [inline README.md:14]: copy/paste typo - should be AWSSNS library

File: tests/agent.test.nut (hunk 1 above)
Comment (ppetrosh) [reply tests/agent.test.nut:26]: See https://github.com/electricimp/AWSSQS/blob/develop/tests/agent.test.nut#L31

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+1. Select `Services` link (on the top left of the page) and them type `IAM` in the search line
+1. Select `IAM Manage User Access and Encryption Keys` item
+1. Select `Policies` item from the menu on the left
+1. Press `Create Policy` button
+1. Press `Select` for `Policy Generator`
+1. On the `Edit Permissions` page do the following
+    1. Set `Effect` to `Allow`
+    1. Set `AWS Service` to `Amazon SNS`
+    1. Set `Actions` to `All Actions`
+    1. Leave `Amazon Resource Name (ARN)` blank
```
Comment (ppetrosh) [inline example/README.md:19]: You can't actually leave this blank. I had to fix the same here https://github.com/electricimp/AWSSQS/tree/develop/example

Comment (betzrhodes) [reply example/README.md:19]: I set up a topic first then did the IAM policy and user steps.

Comment (ppetrosh) [reply example/README.md:19]: exactly it should be reordered

**Comment by betzrhodes** [comment]
new pull request opened on local develop with above issues fixed.

**Pull request closed without merging**